internal/cli/email_templates.go

```diff
@@ -211,11 +212,20 @@ func updateEmailTemplateCmd(cli *cli) *cobra.Command {
 				&inputs.Body,
 				oldTemplate.GetBody(),
 				inputs.Template+".*.liquid",
-				cli.emailTemplateEditorHint,
```

Comment:
Let's remove `emailTemplateEditorHint` as well.

Reply:
This is the only one I can remove safely:)

Reply:
Removed in [`ec270f6` (#603)](https://github.com/auth0/auth0-cli/pull/603/commits/ec270f63792a6f13cc0cd7c453c7b79ee3e69b3a).